cmd/olclient/requests.go

```diff
@@ -0,0 +1,80 @@
+/*
+   ____             _              _                      _____           _                  _
+  / __ \           | |            | |                    |  __ \         | |                | |
+ | |  | |_ __   ___| |     ___  __| | __ _  ___ _ __     | |__) | __ ___ | |_ ___   ___ ___ | |
+ | |  | | '_ \ / _ \ |    / _ \/ _` |/ _` |/ _ \ '__|    |  ___/ '__/ _ \| __/ _ \ / __/ _ \| |
+ | |__| | | | |  __/ |___|  __/ (_| | (_| |  __/ |       | |   | | | (_) | || (_) | (_| (_) | |
+  \____/|_| |_|\___|______\___|\__,_|\__, |\___|_|       |_|   |_|  \___/ \__\___/ \___\___/|_|
+                                      __/ |
+                                     |___/
+
+
+Copyright 2017 - 2019 OneLedger
+*/
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/Oneledger/protocol/client"
+	"github.com/Oneledger/protocol/data/evidence"
+)
+
+var requestsCmd = &cobra.Command{
+	Use:   "requests",
+	Short: "Print out vote requests",
+	Run:   GetVoteRequests,
+}
+
+type RequestsArguments struct {
+	address []byte
```

Comment:
By specifying this address, are we able to filter requests by malicious address? Or are we able to get the full list of requests if leaving address as empty?